cmake/SSGCommon.cmake

```diff
@@ -321,6 +330,7 @@ macro(ssg_build_sce PRODUCT)
             COMMAND ${CMAKE_COMMAND} -E make_directory "${BUILD_CHECKS_DIR}/sce"
             COMMAND ${CMAKE_COMMAND} -E touch "${BUILD_CHECKS_DIR}/sce/metadata.json"
             COMMENT "[${PRODUCT}-content] generating sce/metadata.json"
+            DEPENDS "${CMAKE_CURRENT_BINARY_DIR}/product.yml"
```

Comment:
Is this dependency necessary?

Reply:
Yes, and this is also a reason why the extremely cheap compilation of product is a separate step in the build process.

Reply:
It is not clear to me why it is necessary. The content seems to build without the dependency.

Reply:
Yes, I still don't understand why it is needed. When I removed this dependency, the Ubuntu content with and without SCE was built successfully.
The only difference was the order in which the targets were executed:
Without SCE:
```
[  0%] Built target ubuntu2204-tables
[  5%] [ubuntu2204-content] generating sce/metadata.json
[  5%] Built target generate-internal-ubuntu2204-sce-metadata.json
[ 11%] [ubuntu2204-content] compiling product yaml
[ 16%] [ubuntu2204-content] compiling everything
[ 16%] Built target ubuntu2204-compile-all
```

With SCE:
```
 [  0%] Built target ubuntu2204-tables
[  5%] [ubuntu2204-content] compiling product yaml
[ 10%] [ubuntu2204-content] generating sce/metadata.json
[ 10%] Built target generate-internal-ubuntu2204-sce-metadata.json
[ 15%] [ubuntu2204-content] compiling everything
[ 21%] Built target ubuntu2204-compile-all

```

Reply:
I am sorry, I have missed the exact location, and it is clear that you are absolutely right on this one.